src/components/radio-button/radio-button.jsx

```diff
@@ -290,6 +294,24 @@ const RadioButtonComponent = (
 					{ renderLabel() }
 				</label>
 			) }
+			{ !! info && (
+				<div className="absolute mr-[2px] bottom-1.5 right-3">
+					<Tooltip
+						arrow
+						title={ info?.heading }
+						content={ info?.description }
+						triggers={ [ 'hover', 'focus' ] }
+						placement="top"
+					>
+						<Info
+							className={ cn(
+								'text-text-primary',
+								sizeClassNames[ size ]?.info
+							) }
+						/>
+					</Tooltip>
+				</div>
+			) }
 			<label
 				className={ cn(
 					'absolute mr-[2px] right-3 flex items-center cursor-pointer rounded-full',
```

Comment:
```suggestion
					'absolute mr-0.5 right-3 flex items-center cursor-pointer rounded-full',
```

Reply:
It must not change need 2px always. if use rm/em it changes based on base value.

Reply:
changed